composer.json

```
@@ -24,7 +24,8 @@
         "contao/core-bundle": "^4.4",
         "codefog/contao-haste": "^4.21",
         "codefog/tags-bundle": "^3.0",
-        "doctrine/dbal": "^2.13 || ^3.0"
+        "doctrine/dbal": "^2.13 || ^3.0",
+        "symfony/string": "^5.4"
```

```suggestion
        "symfony/string": "^5.4 || ^6.0"
```

As far as I can see we can be forward compatible here.

---

Makes sense.